source/manual/cdn.html.md

```diff
@@ -28,55 +25,37 @@ Most of the CDN config is versioned and scripted:
 - [CDN configuration](https://github.com/alphagov/govuk-cdn-config/)
 - [CDN config secrets](https://github.com/alphagov/govuk-cdn-config-secrets)
 
-These are deployed to [integration][integration_cdn], [staging][staging_cdn]
-and [production][production_cdn].
+These are deployed to [integration][integration_cdn], [staging][staging_cdn] and [production][production_cdn].
 
-Some configuration isn't scripted, such as logging. The www, bouncer and assets
-services sends logs to S3 and stream them to `monitoring-1`. These logging
-endpoints are configured directly in the Fastly UI. There is
-[documentation](/manual/query-cdn-logs.html) on how to query the CDN logs.
+Some configuration isn't scripted, such as logging. The www, bouncer and assets services sends logs to S3 and stream them to `monitoring-1`. These logging endpoints are configured directly in the Fastly UI. There is [documentation](/manual/query-cdn-logs.html) on how to query the CDN logs.
```

Comment:
```suggestion
Some configuration isn't scripted, such as logging. The www, bouncer and assets services send logs to S3 and stream them to `monitoring-1`. These logging endpoints are configured directly in the Fastly UI. There is [documentation on how to query the CDN logs](/manual/query-cdn-logs.html).
```

Reply:
Much better link - thanks for the suggestion :thumbs: